src/datasources/relay-api/gelato-api.service.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { FakeConfigurationService } from '@/config/__tests__/fake.configuration.service';
+import { FakeCacheService } from '@/datasources/cache/__tests__/fake.cache.service';
+import { CacheDir } from '@/datasources/cache/entities/cache-dir.entity';
+import { GelatoApi } from '@/datasources/relay-api/gelato-api.service';
+import { ILoggingService } from '@/logging/logging.interface';
+import { faker } from '@faker-js/faker';
+import { GelatoRelay } from '@gelatonetwork/relay-sdk';
+import { Hex } from 'viem';
+
+const mockGelatoClient = jest.mocked({
+  sponsoredCall: jest.fn(),
+} as jest.MockedObjectDeep<InstanceType<typeof GelatoRelay>>);
+
+const mockLoggingService = {
+  info: jest.fn(),
+} as jest.MockedObjectDeep<ILoggingService>;
+
+describe('GelatoApi', () => {
+  let target: GelatoApi;
+  let fakeConfigurationService: FakeConfigurationService;
+  let fakeCacheService: FakeCacheService;
+
+  beforeEach(async () => {
+    jest.resetAllMocks();
+
+    fakeConfigurationService = new FakeConfigurationService();
+    fakeCacheService = new FakeCacheService();
+
+    target = new GelatoApi(
+      mockGelatoClient,
+      fakeConfigurationService,
+      fakeCacheService,
+      mockLoggingService,
+    );
+  });
+
+  describe('getRelayCount', () => {
+    it('should return the current count from the cache', async () => {
+      const chainId = faker.string.numeric();
+      const address = faker.finance.ethereumAddress();
+      const cacheDir = new CacheDir(`${chainId}_relay_${address}`, '');
+
```

Comment:
```suggestion
```

Reply:
Removed in 641026000723aeba2b9c4abce2179b1b7d54ecd1.